pkg/services/sqlstore/apikey_test.go

```diff
@@ -27,5 +28,115 @@ func TestApiKeyDataAccess(t *testing.T) {
 			})
 
 		})
+
+		t.Run("Add non expiring key", func(t *testing.T) {
+			cmd := m.AddApiKeyCommand{OrgId: 1, Name: "non-expiring", Key: "asd1", SecondsToLive: 0}
+			err := AddApiKey(&cmd)
+			if err != nil {
+				t.Fatalf("expected to work %v", err)
```

Comment:
Just fyi we have an `github.com/stretchr/testify/assert` util library as dependency so you can use `assert.Nil(err)`.